src/core_plugins/kbn_vislib_vis_types/public/controls/vislib_basic_options.html

```diff
@@ -1,18 +1,19 @@
 <div>
   <div class="vis-option-item">
-    <label>
+    <label id="visualizeBasicSettingsLegendPosition">
       Legend Position
     </label>
     <select
+      aria-labelledby="visualizeBasicSettingsLegendPosition"
       class="form-control"
       ng-model="vis.params.legendPosition"
       ng-options="position.value as position.text for position in vis.type.editorConfig.collections.legendPositions"
     >
     </select>
   </div>
   <div class="vis-option-item">
-    <label>
-      <input type="checkbox" ng-model="vis.params.addTooltip">
+    <label id="visualizeBasicSettingsShowTooltip">
```

Comment:
You shouldn't need this at all. Since the `input` is already inside the `label` they are correctly linked, and you wouldn't need to link them specially for screen readers.